nucypher/blockchain/eth/actors.py

```diff
@@ -383,13 +384,20 @@ def deploy_beneficiary_contracts(self,
                                      emitter: StdoutEmitter = None,
                                      ) -> Dict[str, dict]:
         """
-        The allocation file is a JSON file containing a list of allocations. Each allocation has a:
+        The allocation file is a CSV file containing a list of allocations. Each allocation has a:
```

Comment:
"A CSV or a JSON file", perhaps? Also, judging by the code, the CSV file is recognized by extension, and anything else is considered JSON - perhaps is worth mentioning too.

Reply:
✅